### DISCUSSION TITLE
I try upgrade fremwork 1.12 to 1.13, And have something wrong GRCP.

### DISCUSSION THREAD

**Discussion by Jeniphan** [goravel/goravel#262] [Q&A] [answered]
I try upgrade fremwork from 1.12 to 1.13 I follow document offical. And then I run project use `air` project http is normal run. but have some error log on console `local.error: Run grpc error: host can't be empty`, I know set .env to GRCP_HOST and GRCP_PORT, Then fremwork listening port 3000 is normal and have listening grcp port 3001 (is set in .env) is not normal? Because I try use project is version 1.13 don't have not run grcp listening, If I need not listening grcp port 3001, How I can do? Thanks, Sorry for English language

**Comment by hwbrzzl** [comment]
Can you leave some screenshots here?

**Reply by Jeniphan** [reply]
This error image. If i don't set .env file.
<img width="876" alt="Screenshot 2566-09-22 at 15 57 57" src="https://github.com/goravel/goravel/assets/34912803/94d351ed-ac13-49bf-92af-6f2bbecf424c">
This .env file.
<img width="336" alt="Screenshot 2566-09-22 at 15 59 19" src="https://github.com/goravel/goravel/assets/34912803/27418dfb-e455-49fa-ac01-20423cb2d827">
And If i set .env port and host, It look like this.
<img width="646" alt="Screenshot 2566-09-22 at 16 00 59" src="https://github.com/goravel/goravel/assets/34912803/f44a5a6f-eff9-4ee3-bda5-dfedd2e4db00">

**Reply by hwbrzzl** [reply, marked as answer]
I think you are calling `facades.Grpc()` in the `main.go` file.

**Reply by Jeniphan** [reply]
Aww, I can see. Thanks for support.
